site/en/examples/text_calculator.ipynb

```diff
@@ -11,7 +11,7 @@
     },
```

Comment:
<p>I'm not sure we ever came to a decision about how to handle model names - run-time selection is a foot-gun, but has some other benefits.</p><p> </p><p>I personally prefer the hard-coded approach, but I think it's out of scope for this review so I filed b/284234353 to track.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
got it, thanks for opening the bug!